SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -1125,21 +1125,23 @@ - (nullable SSKProtoDataMessage *)buildDataMessage:(NSString *_Nullable)recipien
 }
 
 - (SSKProtoContentBuilder *)prepareCustomContentBuilder:(SignalRecipient *)recipient {
-    return SSKProtoContent.builder;
-}
-
-- (nullable NSData *)buildPlainTextData:(SignalRecipient *)recipient
-{
-    NSError *error;
     SSKProtoDataMessage *_Nullable dataMessage = [self buildDataMessage:recipient.recipientId];
-    if (error || !dataMessage) {
-        OWSFailDebug(@"could not build protobuf: %@", error);
+    if (!dataMessage) {
+        OWSFailDebug(@"could not build protobuf");
         return nil;
     }
+    
+    SSKProtoContentBuilder *contentBuilder = SSKProtoContent.builder;
+    [contentBuilder setDataMessage:dataMessage];
```

Comment:
Moving this to `prepareCustomContentBuilder` broke friend requests, device linking and session reset because they were overriding this method and not calling `super`... When you make a fundamental change like this you have to test.